src/hooks/eventLogger.ts

```diff
@@ -19,7 +28,26 @@ export const useEventLogger = (): void => {
             }
         }
 
-        const eventLogger = new EventLogger('https://sourcegraph.com')
-        eventLogger?.log('ViewStaticPage', eventArguments, eventArguments)
+        getEventLogger()?.log('ViewStaticPage', eventArguments, eventArguments)
+    })
+}
+
+export const useLogAllLinkClicks = (): void => {
+    useEffect(() => {
+        const links = document.querySelectorAll('a')
+        for (const link of links) {
+            link.addEventListener('click', () => {
+                const eventArguments = {
+                    textContent: link.textContent,
+                    eventKey: link.dataset.rrUiEventKey,
+                    id: link.id,
+                    buttonStyle: link.dataset.buttonStyle,
+                    buttonType: link.dataset.buttonType,
+                    buttonLocation: link.dataset.buttonLocation,
```

Comment:
Since the variables haven't changed yet in the codebase, I would say we'd probably have to follow up with another ticket to update this and I wouldn't want to block you from getting this in. I created #5557 as a follow up that we can tackle at a later date. It's not 100% confirmed, but it something we'd like to do 🙂